Task 9/features/steps/steps_1.js

```diff
@@ -0,0 +1,208 @@
+require("chromedriver");
+const assert = require('assert');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const { Capabilities, Builder, By, Key, until } = require('selenium-webdriver');
+let chrome = Capabilities.chrome();
+const { setDefaultTimeout } = require('@cucumber/cucumber');
+setDefaultTimeout(60 * 10000);
+
+let driver;
+Before(function () {
+  driver = new Builder().withCapabilities(chrome).build();
+})
+
+After(function () {
+  driver.quit();
+});
+
+Given('I visit Kashkick login website', async function () {
+  await driver.get('https://www.kashkick.com/login.php')
+});
+
+
+
+When('I click Header Logo', async function () {
+  await driver.findElement(By.className("img-fluid")).click();
+});
+
+Then('I should be returned on Kashkick main page', async function () {
+  let titleText = await driver.findElement(By.xpath('//*[(contains(text(), "Greatest Reward"))]')).getText();
+        assert.equal('Cash is the Greatest Reward!', titleText)
+});
+
+
+
+
+When('I click Footer Logo', async function () {
+  await driver.findElement(By.className("img-fluid d-block mb-3 lazy")).click()
```

Comment:
create a page object and move there all the driver interactions

Reply:
pushed third commit with some changes, and added a missing data tables example, started to work on page object pattern